test/models/assessment_test.rb

```diff
@@ -37,15 +37,15 @@ def teardown; end
     timestamp_1 = DateTime.now.utc - 5.days
     assessment_1 = create(:assessment, patient: patient, symptomatic: false, created_at: timestamp_1)
     assert_nil patient.symptom_onset
-    assert_in_delta timestamp_1, patient.latest_assessment_at, 1
+    assert_in_delta timestamp_1, patient.latest_assessment.reported_at, 1
```

Comment:
I think this should actually not be changed and `patient.latest_assessment_at` needs to be updated to be based off of `assessment.reported_at` instead of `assessment.created_at`

Reply:
I fixed this and and wrote the migration: https://github.com/SaraAlert/SaraAlert/pull/1154/commits/521ec24c6a299fb5e4ac9b60fe614c74a5c652da

@ngfreiter @sgober please review these changes since I've made quite a few